src/main/java/spireMapOverhaul/zones/Junkyard/powers/JunkGrabPower.java

```diff
@@ -0,0 +1,38 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by FernFlower decompiler)
+//
```

Comment:
I won#t highlight them all, same as with the debug sysouts, but please be sure to remove them